test/utility/misc.c

```diff
@@ -141,11 +141,19 @@ testutil_clean_work_dir(const char *dir)
  *     Get the git top level directory and concatenate the build directory.
  */
 void
-testutil_build_dir(char *buf, int size)
+testutil_build_dir(TEST_OPTS *opts, char *buf, int size)
 {
     FILE *fp;
     char *p;
 
+    /* If a build directory was manually given as an option we can directly
```

Comment:
Nit: Comment should be like:
```
/*
 * Blah blah blah blah blah
 * blah blah
 */
```

Reply:
Thanks for pointing that out! I ran through the dist tooling anyway and it converted it to a single-line comment. Pushed the formatting update :)